packages/android_intent_plus/android/build.gradle

```diff
@@ -46,7 +46,8 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.annotation:annotation:1.7.0'
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.10"))
```

Comment:
This is one of ways to resolve build errors due to different AndroidX packages having different Kotlin versions.

See https://youtrack.jetbrains.com/issue/KT-55297/kotlin-stdlib-should-declare-constraints-on-kotlin-stdlib-jdk8-and-kotlin-stdlib-jdk7

Reply:
That fix from above didn't work on CI, so replaced with another one, which I use in other projects: https://youtrack.jetbrains.com/issue/KT-54136/Duplicated-classes-cause-build-failure-if-a-dependency-to-kotlin-stdlib-specified-in-an-android-project#focus=Comments-27-6583109.0-0